web/styles/settings.css

```diff
@@ -656,11 +652,46 @@ input[type="checkbox"] {
     margin: 10px 0;
 }
 
-.add-new-default-stream-box {
-    & input[type="text"] {
-        padding: 6px;
+#add-default-stream-modal {
+    .dropdown-list-widget {
```

Comment:
I don't think `dropdown-list-widget` class is used for the new dropdown widget. Can you verify?

Reply:
Yeah, it was used in the old dropdown widget. Removed. Thanks.